config/settings.yml

```diff
@@ -33,8 +33,8 @@ EXHIBITS:
   NUM_RESULTS_SHOWN: 3
   QUERY_URL: 'https://exhibits.stanford.edu/search?q=%{q}'
 LIBRARY_WEBSITE:
-  API_URL: 'https://library.stanford.edu/bento?keys=%{q}'
-  QUERY_URL: 'https://library.stanford.edu/search/website?search=%{q}'
+  API_URL: 'https://library.sites-pro.stanford.edu/jsonapi/index/full_site_content?fields[node--stanford_page]=path,title,su_page_description&filter[fulltext]=%{q}&page[limit]=%{max}'
```

Comment:
Will `sites-pro` be in the real URL after launch?

Reply:
Yes. This was something confirmed with Sarah and SWS yesterday. That is the home of the "decoupled" Drupal backend.